src/pro_create.cpp

```diff
@@ -36,19 +37,19 @@ extern bool focus_is_usable_by_ch(struct obj_data *focus, struct char_data *ch);
 void pedit_disp_menu(struct descriptor_data *d)
 {
   CLS(CH);
-  send_to_char(CH, "1) Name: ^c%s^n\r\n", d->edit_obj->restring);
-  send_to_char(CH, "2) Type: ^c%s^n\r\n", programs[GET_DESIGN_PROGRAM(d->edit_obj)].name);
-  send_to_char(CH, "3) Rating: ^c%d^n\r\n", GET_DESIGN_RATING(d->edit_obj));
+  send_to_char(CH, "1) Name: ^c%s^n\r\n", d->edit_matrix_file->name);
+  send_to_char(CH, "2) Type: ^c%s^n\r\n", programs[d->edit_matrix_file->file_type].name);
+  send_to_char(CH, "3) Rating: ^c%d^n\r\n", d->edit_matrix_file->rating);
 
   // Minimum program size is rating^2.
-  int program_size = GET_DESIGN_RATING(d->edit_obj) * GET_DESIGN_RATING(d->edit_obj);
-  if (GET_DESIGN_PROGRAM(d->edit_obj) == SOFT_ATTACK) {
+  int program_size = d->edit_matrix_file->rating ^ 2;
+  if (d->edit_matrix_file->file_type == SOFT_ATTACK) {
     // Attack programs multiply size by a factor determined by wound level.
-    program_size *= attack_multiplier[GET_DESIGN_PROGRAM_WOUND_LEVEL(d->edit_obj)];
-    send_to_char(CH, "4) Damage: ^c%s^n\r\n", GET_WOUND_NAME(GET_DESIGN_PROGRAM_WOUND_LEVEL(d->edit_obj)));
+    program_size *= attack_multiplier[d->edit_matrix_file->attack_damage];
+    send_to_char(CH, "4) Damage: ^c%s^n\r\n", GET_WOUND_NAME(d->edit_matrix_file->attack_damage));
```

Comment:
`attack_damage` is an ambiguous name (it could mean the power `4`, the wound category `D`, or the combined `4D`). Can this be switched to `->wound_category`?